docs/cody/enterprise/model-configuration.mdx

```diff
@@ -515,3 +515,54 @@ The `reasoningEffort` field is only used by reasoning models (those having `reas
 * `openai` provider maps the `reasoningEffort` field value to the [OpenAI `reasoning_effort`](https://platform.openai.com/docs/api-reference/chat/create#chat-create-reasoning_effort) request body value.
 
 </Accordion>
+
+## Google Vertex AI Authentication Configuration
+
+Configure Google Vertex AI authentication using any of the following methods in your `providerOverrides`:
+
+```json
+"providerOverrides": [
+  {
+    "id": "google",
+    "displayName": "Anthropic via Vertex AI",
+    "serverSideConfig": {
+      "type": "google",
+      "endpoint": "https://us-east5-aiplatform.googleapis.com/v1/projects/your-project/locations/us-east5/publishers/anthropic/models"
+
+      // Option 1: Application Default Credentials (recommended - no additional config)
+
+      // Option 2: Explicit credentials file
+      // "googleWorkloadIdentityConfig": {
+      //   // Note: this file must be mounted in the Sourcegraph frontend container!
+      //   "credentialsFile": "/var/secrets/google/credentials.json"
+      // }
+
+      // Option 3: Inline credentials JSON
+      // "googleWorkloadIdentityConfig": {
+      //   "credentialsJSON": "<YOUR BASE64-ENCODED CREDENTIALS JSON>"
+      // }
+
+      // Option 4: Legacy access token
+      // "accessToken": "<YOUR BASE64-ENCODED SERVICE ACCOUNT JSON>"
```

Comment:
does `//` when the doc site renders json? I thought only jsonc allows for comments. But let's try and see